pages/index.js

```diff
@@ -1,12 +1,15 @@
-import Link from "next/link";
+import StoryList from "@/components/StoryList";
+import stories from "@/public/db";
+import { useRouter } from "next/router";
 
 export default function HomePage() {
+  const router = useRouter();
+  function handleStorySelect(story) {
+    router.push(`/stories/${story.id}`);
+  }
```

Comment:
Instead of working with router-push, you could also use `next/link`. As you do not even use `onStorySelect` in your `StoryList`component, I think you can remove the router part and the `handleStorySelect`.

Reply:
done...